db/migrations/add_court_column.py

```diff
@@ -0,0 +1,23 @@
+import os
+import json
+from playhouse.migrate import *
+from db.models import *
+from helpers import get_full_path
+
+HTML_TEXT_FIELDS = ['html_with_citations', 'html', 'html_lawbox', 'html_columbia', 'html_anon_2020', 'plain_text']
```

Comment:
There's a bit of extra copy-pasted code here but since it's a migration, who cares.